app/views/content/trainees/got-into-teaching.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Got Into Teaching"
+description: Celebrate accepting your teacher training place and starting your journey to becoming a teacher.
+content:
```

Comment:
is this content intentionally implemented as two distinct components: a header image and then a title element?
Would it be possible to implement both of these features (image + title) within a single component instead?